napari/_tests/utils.py

```diff
@@ -129,25 +131,15 @@ def check_view_transform_consistency(layer, viewer, transf_dict):
         the transform property (i.e. `scale`, `translate`) and the value
         corresponding to the array of property values
     """
+    if layer.multiscale:
+        return
```

Comment:
I feel like this check should be changed to return True/False/None, or something like that, so that multiscale doesn't just look like it's passed. That seems to be courting trouble to me...

Reply:
I can make `return None` agreed though this feels a little dicey